src/Clients/ThreadClient.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using StreamChat.Models;
+using StreamChat.Rest;
+
+namespace StreamChat.Clients
+{
+    public class ThreadClient : ClientBase, IThreadClient
+    {
+        internal ThreadClient(IRestClient client) : base(client)
+        {
+        }
+
+        private static string Endpoint() => "threads";
```

Comment:
For consistency, please put the endpoint keyword inside the method call. Like here: https://github.com/GetStream/stream-chat-net/blob/master/src/Clients/DeviceClient.cs